File: java/85 Calculator.java
package code;
 class Calculator {
	 int add(int num1,int num2) {
		 return num1+num2;
	 }
	 int add(int num1,int num2,int num3) {
		 return num1+num2+num3;
	 }
	 float add(float num1,float num2) {
		 return num1+num2;
	 }
}
public class Test {
	public static void main(String[] args) {
		Calculator c1=new Calculator();
		System.out.println(c1.add(1, 2));
		System.out.println(c1.add(1, 2,3));
		System.out.println(c1.add(1.4f, 2.4f));
	}
}

File: python/85 Calculator.py
class Calculator:
    def add(self,*num1:int | float) -> int | float:
        sum=0
        for i in num1:
            sum+=i
        return sum

c1=Calculator()
print(c1.add(1,2))
print(c1.add(1,2,3))
print(c1.add(1.2,2.4))